Source/Protocols/Request.swift

```diff
@@ -12,11 +11,24 @@ public protocol Request {
 
 public extension Request where ResponseObject: Decodable {
   func parse(_ data: Data) throws -> ResponseObject {
-    return try JSONDecoder().decode(ResponseObject.self, from: data)
+    let decoder = JSONDecoder()
+    decoder.dateDecodingStrategy = .formatted(DateFormatter.iso8601Full)
```

Comment:
I can see the value in improving our default handling of ISO 8601 date formats, as well as other `JSONDecoder` configuration options. However, I think this could use some more discussion, and would prefer not to tie this to a Swift 5 update. Would you mind separating this change out into a separate PR, or an issue so we can discuss alternative designs?

Have you tried overriding this implementation of `parse(_:)` in your project? Did you run into any issues?

Reply:
Sure, I'll move it to a different PR. I have added this to implementations of `parse(_:)`, but for most project this means overriding every single implementation as the default settings don't properly handle ISO8601 like they're supposed to.